agents/summarizer.py

```
@@ -313,6 +318,127 @@ def generate_using_tgwui(
+                            "type": "step_update",
+                            "phase": {"id": phase_id},
+                            "step": {"id": step_id, "token": snippet, "full_text_len": len(text)},
+                            "token": snippet,
+                        })
+                    last_emit_len = len(text)
+                    last_emit_time = now
+
+                if s and s not in ("tool_call", ""):
+                    break
+
+        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)